server/test/stubs/messages.stub.ts

```diff
@@ -0,0 +1,12 @@
+import { Message } from '../../src/messages/entities/message.entity';
+import { User } from '../../src/users/entities/user.entity';
+
+export const messageStub = (): Message => {
+  return {
+    id: 1,
+    text: 'fakeMessage',
+    created_date: new Date(2021, 11, 6),
+    user: new User(),
```

Comment:
what happens when you create a new entity instance with empty data? should we instead be using the user stub here?

Reply:
On this one, I passed an empty `User`, because if I reference the user stub, it will create a circular dependency